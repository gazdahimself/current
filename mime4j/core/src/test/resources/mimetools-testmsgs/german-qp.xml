<message>
<header>
<field>
Content-Type: text/plain; charset="iso-8859-15"</field>
<field>
Content-Disposition: inline</field>
<field>
Content-Transfer-Encoding: quoted-printable</field>
<field>
MIME-Version: 1.0</field>
<field>
X-Mailer: JaM - Just A Mailer</field>
<field>
Subject: Testnachricht</field>
<field>
Return-Path: &lt;joern@zyn.de&gt;</field>
<field>
Date: Wed, 21 Dec 2005 22:02:44 +0100</field>
<field>
To: joern@zyn.de</field>
<field>
From: =?ISO-8859-15?Q?J=F6rn?= Reder &lt;joern@zyn.de&gt;</field>
</header>
<body>
=0A=
Hallo,=0A=
=0A=
das ist eine Testnachricht mit 8 Bit S=F6nderz=E4ichen, und obendrein noch=
=20=0A=
quoted-printable kodiert.=0A=
=0A=
Gr=FC=DFe,=0A=
=0A=
J=F6rn=0A=

--=20=0A=
 .''`.  J=F6rn Reder &lt;joern@zyn.de&gt;=0A=
: :' :  http://www.exit1.org/ http://www.zyn.de/=0A=
`. `'=0A=
  `-    Debian GNU/Linux -- The power of freedom=0A=
</body>
</message>
