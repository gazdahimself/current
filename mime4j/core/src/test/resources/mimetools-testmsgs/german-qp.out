Content-Type: text/plain; charset="iso-8859-15"
Content-Disposition: inline
Content-Transfer-Encoding: quoted-printable
MIME-Version: 1.0
X-Mailer: JaM - Just A Mailer
Subject: Testnachricht
Return-Path: <joern@zyn.de>
Date: Wed, 21 Dec 2005 22:02:44 +0100
To: joern@zyn.de
From: =?ISO-8859-15?Q?J=F6rn?= Reder <joern@zyn.de>


Hallo,

das ist eine Testnachricht mit 8 Bit S=F6nderz=E4ichen, und obendr=
ein noch 
quoted-printable kodiert=2E

Gr=FC=DFe,

J=F6rn

-- 
 =2E''`=2E  J=F6rn Reder <joern@zyn=2Ede>
: :' :  http://www=2Eexit1=
=2Eorg/ http://www=2Ezyn=2Ede/
`=2E `'
  `-    Debian GNU/Linux -- The powe=
r of freedom
